modin/experimental/xgboost/xgboost.py

```diff
@@ -25,6 +25,23 @@
 LOGGER = logging.getLogger("[modin.xgboost]")
 
 
+def _remove_docstring(obj):
+    """
+    Remove docstring from `obj`.
+
+    Parameters
+    ----------
+    obj : any
+        Object from which need to remove docstring.
+
+    Returns
+    -------
+    `obj` without docstring.
+    """
+    setattr(obj, "__doc__", "")
```

Comment:
```suggestion
    setattr(obj, "__doc__", None)
```

Reply:
If we will set `__doc__ to `None`, inherited docstring will be used.